jp/components/toast.md

```diff
@@ -131,47 +131,13 @@ public show(toast) {
 
 <div class="divider--half"></div>
 
-### API まとめ
+### API リファレンス
 
-以下の表は便利な **igx-toast** コンポーネントの入力、出力、およびメソッドを示します。
+このトピックでは、[`IgxToastComponent`]({environment:angularApiUrl}/classes/igxtoastcomponent.html) を使用と構成方法を説明しました。API の詳細については以下のリンク先を参照してください。
 
-<div class="divider--half"></div>
-
-#### 入力
-
-以下の入力は **igx-toast** コンポーネントで利用できます。
-
-| 名前 | 型 | 説明 |
-| :--- | :--- | :--- |
-| `autoHide` | boolean | `displayTime` 後、Toast を自動的に隠すかどうかを指定します。デフォルトで、`autoHide` は **true** に設定されます。 |
-| `displayTime` | number | Toast が表示される間隔をミリ秒で指定します。デフォルトでは、`displayTime` が 4000 ミリ秒に設定されます。 |
-| `isVisible` | boolean | コンポーネントのビジュアル ステートを指定します。デフォルトで、`isVisible` は **false** に設定されます。 |
-| `message` | string | 表示されるメッセージを指定します。 |
-| `position` | `IgxToastPosition` | Toast の位置を指定します。オプションは `IgxToastPosition.Top`、`IgxToastPosition.Middle`、および `IgxToastPosition.Bottom` です。デフォルトでは、`position` は `IgxToastPosition.Bottom` に設定されます。 |
-
-<div class="divider--half"></div>
-
-#### 出力
-
-以下の出力は **igx-toast** コンポーネントで利用できます。
-
-| 名前 | 説明 |
-| :--- | :--- |
-| `onShowing`  | Toast が表示される前にイベントを発生します。  |
-| `onShown`  | Toast が表示されたときにイベントを発生します。 |
-| `onHiding`  | Toast が非表示される前にイベントを発生します。 |
-| `onHidden`  | Toast が非表示にされたときにイベントを発生します。 |
-
-#### メソッド
-
-以下のメソッドが **igx-toast** コンポーネントで利用できます。
-
-| 名前 | 説明 |
-| :--- | :--- |
-| `show()`  | `autoHide` を **true** に設定した場合、Toast コンポーネントを表示し、`displayTime` 後に非表示にします。 |
-| `hide()`  | Toast コンポーネントを非表示にします。 |
+* [`IgxToastComponent`]({environment:angularApiUrl}/classes/igxtoastcomponent.html)
```

Comment:
Provide Sassdoc reference to the [igx-toast-theme](https://staging.infragistics.local/products/ignite-ui-angular/docs/sass/#function-igx-toast-theme)